jobs/src/test/scala/dcos/metronome/SimulatedScheduler.scala

```diff
@@ -0,0 +1,82 @@
+package dcos.metronome
+
+import akka.actor.{ Cancellable, Scheduler }
+import java.util.concurrent.atomic.AtomicLong
+
+import dcos.metronome.utils.time.FixedClock
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+/**
```

Comment:
this is copypasted from Marathon with just small alterations. I think it is not worth sharing this piece of code - e.g. in a lib or something...

Reply:
Maybe we can create a PR for akka :)